tools/bazar/services/BazarListService.php

```diff
@@ -0,0 +1,244 @@
+<?php
+
+namespace YesWiki\Bazar\Service;
+
+use YesWiki\Bazar\Controller\EntryController;
+use YesWiki\Wiki;
+use YesWiki\Bazar\Field\BazarField;
+use YesWiki\Bazar\Service\EntryManager;
+use YesWiki\Bazar\Service\ExternalBazarService;
+use YesWiki\Bazar\Service\FormManager;
+
+class BazarListService
+{  
+    public function __construct(Wiki $wiki, EntryManager $entryManager, EntryController $entryController,
+                                ExternalBazarService $externalBazarService, FormManager $formManager)
+    {
+      $this->wiki = $wiki;
+      $this->entryManager = $entryManager;
+      $this->entryController = $entryController;
+      $this->externalBazarService = $externalBazarService;
+      $this->formManager = $formManager;
+    }
+
+    public function getForms($options) : array
```

Comment:
```suggestion
    public function getForms(array $arguments) : array
```

très bonne idée cette méthode, toutefois pour ne pas trop nous confondre, je préfère conserver le terme `$arguments`